src/main/kotlin/com/grepp/nbe1_3_team04/team/api/request/TeamCreateRequest.kt

```diff
@@ -0,0 +1,19 @@
+package com.grepp.nbe1_3_team04.team.api.request
+
+import com.grepp.nbe1_3_team04.team.service.request.TeamDefaultServiceRequest
+import jakarta.validation.constraints.NotNull
+
+data class TeamCreateRequest(
+    @field:NotNull(message = "팀 명은 필수입니다.")
+    val name: String?,
+    val description: String?,
+    val location: String?
+) {
+    fun toServiceRequest(): TeamDefaultServiceRequest {
+        return TeamDefaultServiceRequest(
+            name!!,
+            description = description ?: "",
+            location = location ?: ""
```

Comment:
```suggestion
            description = description,
            location = location
```
TeamDefaultServiceRequest의 해당 필드를 nullable하게 설정해주면 이렇게 해도 빨간줄 안뜨실겁니다!